sunpy/map/mapbase.py

```diff
@@ -193,7 +193,32 @@ def __init_subclass__(cls, **kwargs):
         if cls.__doc__ is None:
             # Set an empty string, to prevent an error adding None to str in the next line
             cls.__doc__ = ''
-        cls.__doc__ += textwrap.indent(_notes_doc, "    ")
+
+        def notes_fix(notes_doc, datadoc):
+            existing_notes_pos = datadoc.find('Notes\n    -----')
+            existing_notes_pos2 = notes_doc.find('Notes\n-----')
```

Comment:
Why does one have 4 spaces and the other doesnt?

Reply:
This is simply because the existing_notes_pos is present in Notes section which is in the a Class, so even after new line there is some indentation space there. And exisiting_notes_pos2 is present outside any class in the mapbase.py file